argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc

```diff
@@ -36,6 +36,16 @@ include::modules/gitops-creating-rolloutmanager-custom-resource.adoc[leveloffset
 // Deleting a RolloutManager custom resource
 include::modules/gitops-deleting-rolloutmanager-custom-resource.adoc[leveloffset=+1]
 
+// Argo Rollouts CLI Overview
+include::modules/gitops-argo-rolouts-cli-ovr.adoc[leveloffset=+1]
+
+// Installing Argo Rollouts CLI using Linux
+include::modules/gitops-installing-argo-rollouts-cli-using-linux.adoc[leveloffset=+1]
```

Comment:
```suggestion
include::modules/gitops-installing-argo-rollouts-using-linux.adoc[leveloffset=+2]
```
ID must be the same as the file name: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

Reply:
Fixed.